apps/events/models.py

```diff
@@ -17,3 +18,6 @@ class Event(models.Model):
 
     def __str__(self):
         return "Event: {" + self.name + "}"
+
+    def get_absolute_url(self):
+        return "https://cosmostue.nl" + reverse("cosmos_events:events-view", kwargs={"pk": self.pk})
```

Comment:
The server should be independent of the host domain. Refer to other instances of `get_absolute_url()` in the repo.